helm/prometheus-agent/templates/_helpers.tpl

```diff
@@ -3,35 +3,33 @@
 Expand the name of the chart.
 */}}
 {{- define "name" -}}
-{{- .Chart.Name | trunc 63 | trimSuffix "-" -}}
+{{- $.Chart.Name | trunc 63 | trimSuffix "-" -}}
```

Comment:
What does the dollar do?

Reply:
$ refers to the root scope of the chart as per https://helm.sh/docs/chart_template_guide/control_structures/#modifying-scope-using-with.

I fixed the usage only where this is necessary, i.e. in the loop itself